src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleNoSlow.kt

```diff
@@ -94,6 +95,30 @@ object ModuleNoSlow : Module("NoSlow", Category.MOVEMENT) {
             }
 
 
+        }
+
+        object Grim : Choice("Grim") {
+
+            override val parent: ChoiceConfigurable
+                get() = modes
+
+            val onNetworkTick = handler<PlayerNetworkMovementTickEvent> { event ->
+                if (event.state == EventState.PRE) {
+                    if (mc.player!!.isUsingItem) {
+                        if (mc.player!!.getActiveHand() === Hand.OFF_HAND) {
+                            mc.player!!.networkHandler.sendPacket(UpdateSelectedSlotC2SPacket(
+                                mc.player!!.inventory.selectedSlot % 8 + 1))
+                            mc.player!!.networkHandler.sendPacket(UpdateSelectedSlotC2SPacket(
+                                mc.player!!.inventory.selectedSlot))
+                        } else {
+                            mc.player!!.networkHandler.sendPacket(PlayerInteractItemC2SPacket(
+                                Hand.OFF_HAND, 0))
```

Comment:
For mc.player!! use player. instead.
For mc.player!!.networkHandler use network. instead

Reply:
> For mc.player!! use player. instead. For mc.player!!.networkHandler use network. instead

this kid stole my code, yeah it was coded really badly lol